roxie/ccd/ccdmain.cpp

```diff
@@ -820,6 +822,9 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         blobCacheMB = topology->getPropInt("@blobCacheMem", 0);
         setBlobCacheMem(blobCacheMB * 0x100000);
 
+        unsigned __int64 affinity = topology->getPropInt64("@affinity", 0);
+        updateAffinity(affinity);
```

Comment:
Bah. I want the control:setAffinity to throwUnimplemented, but not the startup code.